src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,22 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+
+  width: catalog-width(
+    $card-width,
+    $space-between-cards,
+    $cards-in-row,
+    $catalog-paddings
+  );
+
+  padding: 50px $catalog-paddings 0;
```

Comment:
$catalog-paddings is not clear variable. Why is it applying only for left and right sides?